### ISSUE TITLE
Add file load exclusion for Memory.json / never save the file

### ISSUE THREAD

**Issue by sir-wilhelm** [sir-wilhelm/SmartHunter#19]
With the dynamic checking / loading of memory I'm not sure we even need to save it anymore.

**Comment by sir-wilhelm** [comment]
This will also remove/reduce the
```
[2020-02-18 04:57:51] Warning: Memory.json differs from autogenerated version.
[2020-02-18 04:57:51] Memory.json will be renamed to custom_Memory.json and recreated...
[2020-02-18 04:57:51] This can be disabled by setting ["UseCustomData": true,] in Config.json
```
spam on launch.

**Comment by sir-wilhelm** [comment]
Fixed in 1.4.1.0
